ttnn/cpp/ttnn/operations/ccl/line_all_gather/device/line_all_gather_op.cpp

```diff
@@ -2,6 +2,8 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+#include <ranges>
```

Comment:
Where are the ranges being used?

Reply:
yes, you're right. It's not needed in the latest implementation so I removed it. Thanks!